scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/JobInfoView.java

```diff
@@ -62,25 +55,24 @@
 
     protected SchedulerController controller;
 
-    /** generic information table */
-    private ListGrid genericInformationGrid = null;
-
-    /** Generic information column factory */
-    private GenericInformationColumnsFactory genericInformationColumnsFactory;
+    /** Generic information label text */
+    private static final String GENERIC_INFORMATION_LABEL_TEXT = "Generic Information";
 
     /** Generic information label text */
```

Comment:
I don't see why. Since I used twice the same object for both variables and GI i refactored the code